bundles/ranvier-commands/commands/look.js

```diff
@@ -1,17 +1,63 @@
 'use strict';
 
+const leftPad = require('left-pad');
+
 module.exports = (srcPath) => {
   const Broadcast = require(srcPath + 'Broadcast');
   const CommandParser = require(srcPath + 'CommandParser').CommandParser;
   const Item = require(srcPath + 'Item');
   const ItemType = require(srcPath + 'ItemType');
   const Logger = require(srcPath + 'Logger');
 
+  function getCompass(player) {
+    const room = player.room;
+
+    const exitMap = new Map();
+    exitMap.set('east', 'E');
```

Comment:
It's possible to chain `Map.set` to make this a bit less repetitive. 

In addition, while a `Map` is a good candidate for containing values that you won't know at runtime, we know all of the cardinal directions at runtime. So this might not be the best choice here? Not sure though since custom exits could exist as well -- but then how would they be rendered by the compass? Or would they?